Tranning/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="rlt" translate="label" sortOrder="300">
+            <label>RLTSquare</label>
+        </tab>
+        <section id="email" translate="label" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
+            <class>separator-top</class>
+            <label>RLTSquare Email Demo</label>
+            <tab>rlt</tab>
+            <resource>RLTSquare_Tranning::config_emaildemo</resource>
+            <group id="demo" translate="label" type="text" sortOrder="2" showInDefault="1" showInWebsite="1" showInStore="1">
+                <label>Please Choose Your Email Template</label>
+                <field id="template" translate="label" type="select" sortOrder="3" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
Overlooked logic - we should have a config.xml here to set default value to out custom email template.